lib/page/country.rb

```diff
@@ -0,0 +1,9 @@
+module Page
+  class Country
+    attr_reader :country, :title
+    def initialize(country, all_countries)
+      @country = all_countries.find { |c| c[:url] == country }
+      @title = "EveryPolitician: #{@country[:name]}" if @country
```

Comment:
Why are you cramming all this logic into `initialize`? These seem like they should just be normal methods.